content/ react-security-vulnerabilities/index.md

```diff
@@ -0,0 +1,227 @@
+---
+title: "React Security Vulnerabilities and How to Fix/Prevent them"
+date: "2021-12-21"
+coverImage: "react-security.png"

```

Comment:
In the actual image, use title case instead of all uppercase.

Also, give some space in the top so that it won't be cut off in social posts